README.md

```diff
@@ -25,7 +31,8 @@ Example usage:
 ```python
 from ops.charm import CharmBase
 from ops.main import main
-from oci_image import OCIImageResource, OCIImageResourceError
+from src.oci_image import OCIImageResource, OCIImageResourceError
```

Comment:
I really don't like moving this into the `src` namespace, especially if we're going to put it up on PyPI.  We might consider putting it under the `ops.lib` namespace and set it up to match the use of [`ops.lib.use()`](https://github.com/canonical/operator/blob/master/ops/lib/__init__.py#L42), although I'm still on the fence as to what value that brings to the table over standard Python package management.

Reply:
I could make it to PyPI if you dont have time to. What I am wondering is why would having the `src` directory be a problem for PyPI? Would it be because of it being considered a subpackage. `setuptools` has a `find_packages()` method. I any case I reverted this change.

Reply:
I do not know how the `ops.lib.use` work but a nice thing to consider is the versioning we can have easily with PyPI.